ClosedXML/Excel/CalcEngine/Functions/MathTrig.cs

```diff
@@ -129,7 +130,34 @@ private static object Atan2(List<Expression> p)
 
         private static object Ceiling(List<Expression> p)
         {
-            return Math.Ceiling(p[0]);
+            double number = p[0];
+            double significance = 1;
+            if (p.Count > 1)
+                significance = p[1];
```

Comment:
Since the number of arguments is exactly two this can be simplified to `double significance = p[1];`

Reply:
Good catch. I implemented your recommendation.